src/realm/obj.cpp

```diff
@@ -935,7 +935,7 @@ Obj::FatPath Obj::get_fat_path() const
     return result;
 }
 
-FullPath Obj::get_path() const
+FullPath Obj::get_path() const noexcept
```

Comment:
I am not sure this method can be marked `noexcept`.  I haven't checked all our paths in our code base, but `vector<T>::push_back`  can throw.

Reply:
noexcept removed.